engine/src/main/java/org/terasology/rendering/dag/StateChange.java

```diff
@@ -22,4 +22,7 @@
     StateChange getDefaultInstance();
 
     RenderPipelineTask generateTask();
+
+    // TODO: investigate a better way?
+    boolean compare(StateChange stateChange);
```

Comment:
Sure: you could rename it to: 
- isEqualTo
- isJustLike
- isActuallyIdenticalTo

=D

My favourite is "isJustLike". =)